src/mysky/connector.ts

```diff
@@ -0,0 +1,55 @@
+import { Connection, ParentHandshake, WindowMessenger } from "post-me";
+import { createIframe, defaultHandshakeAttemptsInterval, defaultHandshakeMaxAttempts } from "skynet-interface-utils";
+
+import { SkynetClient } from "../client";
+
+export type CustomConnectorOptions = {
+  handshakeMaxAttempts?: number;
+  handshakeAttemptsInterval?: number;
+};
+
+const defaultConnectorOptions = {
+  handshakeMaxAttempts: defaultHandshakeMaxAttempts,
+  handshakeAttemptsInterval: defaultHandshakeAttemptsInterval,
+};
+
+export class Connector {
```

Comment:
this connector will only set up the child iframe and perform the parent handshake. Is it correct to say that it's up to the DAC developers to be aware of this and use `post-me` and pass in the connection and perform the hanshake. Or is this something we want to abstract from those  devs as well?

Reply:
We want to abstract it eventually.